website/docs/cdktf/cli-reference/cli-configuration.mdx

```diff
@@ -30,7 +30,7 @@ Commands:
   cdktf destroy [OPTIONS] <stacks..>  Destroy the given stacks
   cdktf diff [stack] [OPTIONS]        Perform a diff (terraform plan) for the given stack                                                                                                         [aliases: plan]
   cdktf list [OPTIONS]                List stacks in app.
-  cdktf login                         Retrieves an API token to connect to Terraform Cloud.
+  cdktf login                         Retrieves an API token to connect to HCP Terraform.
```

Comment:
```suggestion
  cdktf login                         Retrieves an API token to connect to Terraform Cloud.
```

In the tutorial updates, we had decided to leave the CLI output unaltered. Do we want to do the same in the docs?